tests/stub/iteration/test_result_list.py

```diff
@@ -0,0 +1,210 @@
+import nutkit.protocol as types
+from tests.shared import get_driver_name
+
+from ._common import IterationTestBase
+
+
+class TestResultSingle(IterationTestBase):
+
+    required_features = (types.Feature.BOLT_4_0,
+                         types.Feature.API_RESULT_LIST)
+
+    def _assert_not_exactly_one_record_error(self, error):
+        self.assertIsInstance(error, types.DriverError)
+        driver = get_driver_name()
+        if driver in ["python"]:
+            self.assertEqual("<class 'ToBeDecided'>",
```

Comment:
Are you going to update this later on?

Reply:
The whole method is a copy-paste left-over from `tests/stub/iteration/test_result_single.py`.